tsdb/agent/db_test.go

```diff
@@ -414,7 +488,6 @@ func TestPartialTruncateWAL(t *testing.T) {
 	)
 
 	opts := DefaultOptions()
-	opts.TruncateFrequency = time.Minute * 2
```

Comment:
note to reviewers: with `-race` and the additional samples, the test sometimes runs longer than 2 minutes and this interferes.